core-services/image-mirroring/knative/mapping_knative_v0_14_quay

```diff
@@ -33,3 +33,5 @@ registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-test-se
 registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-test-sendevents quay.io/openshift-knative/sendevents:v0.14.0
 registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-test-heartbeats quay.io/openshift-knative/heartbeats:v0.14.0
 registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-test-recordevents quay.io/openshift-knative/recordevents:v0.14.0
+registry.svc.ci.openshift.org/openshift/knative-v0.14.0:kn quay.io/openshift-knative/kn:v0.14.0
```

Comment:
```suggestion
registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-client quay.io/openshift-knative/kn:v0.14.0
```

this value should match `to` value for this image
https://github.com/openshift/release/pull/8509/files#diff-5876dfcd0f02a4ff1ae9970592eb499cR22

the src image co-ordinates in the mirror config are formed as
```
{REGISTRY}/{NAMESPACE}/{PROMOTION}:{TO}
```

see for example in core-services/image-mirroring/knative/mapping_knative_v0_13_quay